mmv1/templates/terraform/constants/vmwareengine_private_cloud.go.erb

```diff
@@ -0,0 +1,33 @@
+func vmwareenginePrivateCloudStandardTypeDiffSuppressFunc(_, old, new string, d *schema.ResourceData) bool {
+	if (old == "STANDARD" && new == "") || (old == "" && new == "STANDARD") {
+		return true
+	}
+    if (isMultiNodePrivateCloud(d) && old == "TIME_LIMITED" && new == "STANDARD") {
+        log.Printf("[DEBUG] Multinode Private Cloud found, facilitating TYPE change to STANDARD")
```

Comment:
Can you give more context as to why a type diff from `TIME_LIMITED` -> `STANDARD` needs to be suppressed with Multinode? Also, is this related or separate to delayed deletion?

Reply:
I also see that `type` is set to `ignore_read`, so at least it will remain consistent. Thanks for the context.